src/dsl.jl

```diff
@@ -914,68 +930,58 @@ end
 # Function for creating a Reaction structure (used by the @reaction macro).
 function make_reaction(ex::Expr)
 
-    # Handle interpolation of variables
+    # Handle interpolation of variables in the input.
     ex = esc_dollars!(ex)
 
-    # Parses reactions, species, and parameters.
+    # Parses reactions. Extracts species and paraemters within it.
     reaction = get_reaction(ex)
-    species, parameters = extract_species_and_parameters!([reaction], [])
+    species, parameters = extract_sps_and_ps([reaction], [])
 
-    # Checks for input errors.
-    forbidden_symbol_check(union(species, parameters))
```

Comment:
This check is already carried out in `Reaction()`, so no longer needed in the DSL.